test/units/testsuite-81.network-generator.sh

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/env bash
```

Comment:
Oh, please merge this into test/test-network-generator-conversion.sh.

Reply:
Hah, I completely forgot about this one. Should I follow its format and create the .input & .expected files/directories for each combination, or is generating the stuff on the fly fine? As that would be a _lot_ of files otherwise.

Reply:
Both ways fine. Feel free to select one you'd like.